build.sbt

```diff
@@ -159,7 +159,7 @@ lazy val dockerSettings = Seq(
   Docker / packageName := "adopttapir",
   dockerUsername := Some("softwaremill"),
   dockerUpdateLatest := true,
-  Docker / publishLocal := (Docker / publishLocal).dependsOn(copyWebapp).value,
+  Docker / publish := (Docker / publish).dependsOn(copyWebapp).value,
```

Comment:
We probably still want for `publishLocal` to depend on `copyWebapp`. Otherwise we won't get webapp in images created but not published to remote.